webpack.config.js

```diff
@@ -4,10 +4,10 @@ var phaser = path.join(pathToPhaser, 'dist/phaser.js');
 var webpack = require("webpack");
 
 module.exports = env => {
-    var isDebug = (env && env.NODE_ENV === "dev");
+    var isDebug = (env && (env.NODE_ENV === "dev" || env.NODE_ENV === "test"));
 
     return {
-        entry: './src/game.ts',
+        entry: (env && env.NODE_ENV === "test") ? './test/test.ts' : './src/game.ts',
         output: {
             path: path.resolve(__dirname, 'dist'),
```

Comment:
Leaving a note for myself to ensure that bundle size is minimized for both prod and test environments after these changes.

Reply:
No need to address this any further right now.